waku/factory/node_factory.nim

```diff
@@ -414,7 +414,11 @@ proc startNode*(
 
   # retrieve px peers and add the to the peer store
   if conf.peerExchangeNode != "":
-    let desiredOutDegree = node.wakuRelay.parameters.d.uint64()
+    let desiredOutDegree =
+      if not node.wakuRelay.isNil():
+        node.wakuRelay.parameters.d.uint64()
+      else:
+        DefaultPXNumPeersReq
```

Comment:
```suggestion
    var desiredOutDegree = DefaultPXNumPeersReq
    if not node.wakuRelay.isNil() and node.wakuRelay.parameters.d.uint64() > 0:
      desiredOutDegree = node.wakuRelay.parameters.d.uint64()        
```
WDYT, would that make sense?

Reply:
Ooh that's nice! More concise :))

Just applied it! Thank you!